fix: Copy changes

deppp opened the pull request:
This are the copy changes, removing doc links, and changing the language in a few places.

### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)



#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend



### Describe the reason for change
_(link to issue, supportive screenshots etc.)_



#### What does this fix?
_(if this is a bug fix)_



#### What is the new behavior?
_(if this is a breaking or feature change)_



#### What is the current behavior?
_(if this is a breaking or feature change)_



#### What libraries were added/updated?
_(list all with version changes)_



#### Does this change affect performance?
_(if so describe the impacts positive or negative)_



#### Does this change affect security?
_(if so describe the impacts positive or negative)_



#### What alternative approaches were there?
_(briefly list any if applicable)_



#### What feature flags were used to cover this change?
_(briefly list any if applicable)_



### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)



### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit



### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

Commit:
changing the copy back and providing more info

Commit:
changing the copy

Commit:
fixing structure

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8611807449

hlomzik commented on web/apps/labelstudio/src/pages/Settings/AnnotationSettings.js:
```diff
@@ -57,23 +57,13 @@ export const AnnotationSettings = () => {
                 description={
                   <span>
-                    Enable and select which set of predictions to use for prelabeling.
-                    Predictions will be pre-loaded in {" "}
```
Looks like a helpful info given the diversity of affected workflows (and I believe some were missing here anyway) + links to documentation.
Can we maybe rephrase this without removal? Why is it better to not have this info at all?

hlomzik commented on web/apps/labelstudio/src/pages/Settings/AnnotationSettings.js:
```diff
@@ -57,23 +57,13 @@ export const AnnotationSettings = () => {
                 description={
                   <span>
-                    Enable and select which set of predictions to use for prelabeling.
-                    Predictions will be pre-loaded in {" "}
-                    <a
-                      style={{ color: "rgb(105 129 185)" }}
-                      target="_blank" href="https://labelstud.io/guide/labeling.html#Choose-which-tasks-to-label"
-                    >Label&nbsp;All&nbsp;Tasks</a>{" "}
-                    and {" "}
-                    <a
-                      style={{ color: "rgb(105 129 185)" }}
-                      target="_blank" href="https://labelstud.io/guide/get_started#Interface"
-                    >Quick View</a>.
```
Looks like a helpful info given the diversity of affected workflows (and I believe some were missing here anyway, like action in DM to convert predictions into annotations?) + links to documentation.
Can we maybe rephrase this without removal? Why is it better to not have this info at all?

deppp replied:
The state of those linked docs is not great, and it would create more confusion that explain much. We first need to improve those sections, before we include it into the product. Overall, its simplicity.

hlomzik commented on web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js:
```diff
@@ -135,30 +135,22 @@ export const MachineLearningSettings = () => {
+            </Description>
+            <Description style={{ marginTop: 0, display: "flex", gap: "5px" }} className="description_alert_warning">
+              <IconInfo width="20" height="20" />
+              If you want to use the model predictions for prelabeling, please configure this in the Annotation settings.
```
Link to Annotation settings would be helpful as we discussed

deppp replied:
Yes, I'm very supportive if you can implement that :)

hlomzik commented on web/apps/labelstudio/src/components/Description/Description.styl:
```diff
@@ -22,3 +22,10 @@
     &_large
       margin 40px 0
+
+  &_alert
+   &_warning
+     background-color rgb(255, 251, 230);
+     border 1px solid rgb(255, 229, 143);
+     border-radius 8px
+     padding 10px
```
ident and missing trailing new line
```suggestion
  &_alert
    &_warning
      background-color rgb(255, 251, 230);
      border 1px solid rgb(255, 229, 143);
      border-radius 8px
      padding 10px

```

deppp committed:
Update web/apps/labelstudio/src/components/Description/Description.styl

Co-authored-by: hlomzik <hlomzik@gmail.com>

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8619449452

hlomzik commented:
Closed in favor of https://github.com/HumanSignal/label-studio/pull/5712
